### ISSUE TITLE
The tag option to balena build doesn't seem to work

### ISSUE THREAD

**Issue by CameronDiver** [balena-io/balena-cli#1570]
I can't seem to get the tag variable to be respected by the `balena build` command:
```
[10:49] util-containers/network git:(master) ✗ λ balena build -t test-image -A amd64 -d test .
[Info]    Creating default composition with source: /home/cameron/Projects/util-containers/network
[Info]    Building for amd64/test
[Build]   Built 1 service in 0:33
[Build]   main Image size: 193.09 MB
[Success] Build succeeded!
[10:49] util-containers/network git:(master) ✗ λ docker images
REPOSITORY                 TAG                 IMAGE ID            CREATED             SIZE
network_main               latest              43a997b14d68        11 seconds ago      202MB
balenalib/amd64-debian     latest              328983ff8812        29 hours ago        113MB
```

- **balena CLI version:** 11.18.3
- **Operating system version:** Arch
- **32/64 bit OS and processor:** 64bit intel
- **Install method:** npm
- **If npm install, Node.js and npm version:**
```
[10:51] util-containers/network git:(master) ✗ λ node -v
v10.16.3
[10:51] util-containers/network git:(master) ✗ λ npm -v
6.9.0
```
---

*Please keep in mind that we try to use the issue tracker of this repository for specific bug
reports & CLI feature requests. General & troubleshooting questions are encouraged to be posted to
the [balena forums](https://forums.balena.io), which are monitored by balena's support team and
where the community can both contribute and benefit from the answers.*

*Before submitting this issue please check that this issue is not a duplicate. If there is another
issue describing the same problem or feature please add your information to the existing issue's
comments.*

**Comment by pdcastro** [comment]
Related to:
* #825 [Build command ignores --tag argument](https://github.com/balena-io/balena-cli/issues/825)
* #1018 [Deploy command ignores --tag argument](https://github.com/balena-io/balena-cli/issues/1018)

**Comment by CameronDiver** [comment]
My bad, I didn't search well enough. I'm going to close this issue and we can track it with the other - sorry for the noise!

**Comment by pdcastro** [comment]
No problem, I have bumped up that issue's priority in the "[ordered backlog](https://github.com/balena-io/balena-cli/milestone/2)" as a result of this new report - so thanks anyway! I usually don't close issues as a duplicate because sometimes this leads to more people creating more duplicates, as closed issues tend to be ignored when people search for them (there is the assumption that a closed issue has been resolved...). For as long as the issues are linked, a single PR can close several duplicate issues at once.
